src/sentry/seer/similarity/utils.py

```diff
@@ -159,6 +159,45 @@
         "java-spring",
         "java-spring-boot",
         "perl",
+        # Remaining platforms
+        "apple",
+        "apple-ios-profiling-onboarding-1-install",
+        "apple-ios-profiling-onboarding-4-upload",
+        "apple-macos",
+        "c",
```

Comment:
Is c not native? Same for minidump.

Reply:
I honestly don't know. I will comment it out.

Reply:
Okay, sorry. I should be less circumspect. 😛  C and minidump are native, and we shouldn't handle them yet.